models/sudoswap/ethereum/sudoswap_ethereum_events.sql

```diff
@@ -39,7 +39,7 @@ WITH
             WHERE call_success = true
             {% if is_incremental() %}
             -- this filter will only be applied on an incremental run. We only want to update with new swaps.
-            AND call_block_time >= (select max(block_time) from {{ this }})
+            AND call_block_time >= date_trunc("day", now() - interval '1 week')
```

Comment:
fwiw this is being edited in another open PR too, but we can just ensure via merge conflicts to have it in place

Reply:
Ok good to know 👍